### ISSUE TITLE
f31:: AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by adriansev** [FrostyX/tracer#150]
So, i just got this on a fedora 31 system:
```
Running transaction
  Preparing        :                                                                                                                                             1/1
  Installing       : libdb-cxx-5.3.28-38.fc31.x86_64                                                                                                             1/4
  Installing       : nordugrid-arc-6.6.0-1.fc31.x86_64                                                                                                           2/4
  Installing       : nordugrid-arc-plugins-needed-6.6.0-1.fc31.x86_64                                                                                            3/4
  Installing       : nordugrid-arc-client-6.6.0-1.fc31.x86_64                                                                                                    4/4
  Running scriptlet: nordugrid-arc-client-6.6.0-1.fc31.x86_64                                                                                                    4/4
  Verifying        : nordugrid-arc-6.6.0-1.fc31.x86_64                                                                                                           1/4
  Verifying        : nordugrid-arc-client-6.6.0-1.fc31.x86_64                                                                                                    2/4
  Verifying        : nordugrid-arc-plugins-needed-6.6.0-1.fc31.x86_64                                                                                            3/4
  Verifying        : libdb-cxx-5.3.28-38.fc31.x86_64                                                                                                             4/4
Tracer:
 Call to Tracer API ended unexpectedly:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 66, in transaction
    tracer = TracerFacade(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 79, in __init__
    self.apps = self.get_apps(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 84, in get_apps
    return query.from_packages(packages).now().affected_applications().get()
  File "/usr/lib/python3.7/site-packages/tracer/query.py", line 72, in get
    return self._method(**self._kwargs)
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 126, in trace_affected
    if not self._applications.find('kernel').ignore and self._has_updated_kernel():
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 134, in _has_updated_kernel
    running = System.running_kernel_package()
  File "/usr/lib/python3.7/site-packages/tracer/resources/system.py", line 119, in running_kernel_package
    return System.package_manager().find_package(System.kernel_package_name(), os.uname()[2])
  File "/usr/lib/python3.7/site-packages/tracer/resources/PackageManager.py", line 67, in find_package
    return self.package_managers[0].find_package(pkg_name, search)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 119, in find_package
    self._load_package_info_from_hdr(package, hdr)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 201, in _load_package_info_from_hdr
    package.description = hdr[rpm.RPMTAG_SUMMARY].decode()
AttributeError: 'str' object has no attribute 'decode'

Please visit https://github.com/FrostyX/tracer/issues and submit the issue. Thank you
We apologize for any inconvenience



```

**Comment by FrostyX** [comment]
Hello @adriansev,
you seem to be using an older version of tracer. Please upgrade to 0.7.4

See the changelog, the issue was fixed there
https://github.com/FrostyX/tracer/releases/tag/tracer-0.7.4-1

**Comment by FrostyX** [comment]
Did the new version fix the issue for you, @adriansev?

**Comment by FrostyX** [comment]
I believe the issue should be fixed since 0.7.4, please reopen if I am wrong and it still occurs.
